api/base/serializers.py

```diff
@@ -13,6 +12,27 @@
 from api.base.exceptions import InvalidQueryStringError, Conflict
 
 
+def format_relationship_links(related_link, self_link, rel_meta, self_meta):
+
+    ret = {
+        'links': {
+            'related': {
+                'href': related_link,
+                'meta': rel_meta
+            },
+            'self': {
+                'href': self_link,
+                'meta': self_meta
+            }
+        }
+    }
+
+    # Not currently returning self links
+    if not ret['links']['self']['href']:
```

Comment:
I have no strong preference for either of the above, though I expect option 1 would be slightly less impactful on the codebase.